pkg/db/task.go

```diff
@@ -0,0 +1,126 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+type Task struct {
+	ID      string `json:"id,omitempty"`
+	Date    string `json:"date"`
+	Title   string `json:"title"`
+	Comment string `json:"comment"`
+	Repeat  string `json:"repeat"`
+}
+
+func AddTask(task *Task) (int64, error) {
+	var id int64
+
+	query := `INSERT INTO scheduler (date, title, comment, repeat) VALUES (?, ?, ?, ?)`
+	res, err := DB.Exec(query, task.Date, task.Title, task.Comment, task.Repeat)
+	if err == nil {
+		id, err = res.LastInsertId()
+	}
+	return id, err
+}
+
+func Tasks(limit int) ([]*Task, error) {
+	rows, err := DB.Query("SELECT id, date, title, comment, repeat FROM scheduler ORDER BY date LIMIT ?", limit)
+	if err != nil {
+		return nil, fmt.Errorf("Ошибка запроса SELECT к БД: %v", err)
+	}
+	defer rows.Close()
+
+	var tasks []*Task
+
+	for rows.Next() {
+		var task Task
+		err := rows.Scan(&task.ID, &task.Date, &task.Title, &task.Comment, &task.Repeat)
+		if err != nil {
+			return nil, fmt.Errorf("Ошибка при сканировании: %v", err)
+		}
+		tasks = append(tasks, &task)
+	}
+	if err := rows.Err(); err != nil {
+		return nil, err
+	}
+
+	if tasks == nil {
+		tasks = []*Task{}
+	}
+	return tasks, nil
+}
+
+func GetTask(id string) (*Task, error) {
+	query := "SELECT id, date, title, comment, repeat FROM scheduler WHERE id = ?"
+
+	row := DB.QueryRow(query, id)
+
+	var task Task
+
+	err := row.Scan(&task.ID, &task.Date, &task.Title, &task.Comment, &task.Repeat)
+	if err == sql.ErrNoRows {
+		return nil, fmt.Errorf("Не найдена запись с таким id: %v", err)
```

Comment:
мы и так уже знаем что за ошибка
```suggestion
		return nil, fmt.Errorf("Не найдена запись с таким id: %d",task.ID)
```